camisole/isolate.py

```diff
@@ -74,6 +74,26 @@ async def communicate(cmdline, data=None, **kwargs):
 }
 
 
+class IsolateInternalError(RuntimeError):
+    def __init__(self, command, isolate_stdout, isolate_stderr, message=None):
+        self.command = command
+        self.isolate_stdout = isolate_stdout
+        self.isolate_stderr = isolate_stderr
+
+        if message is None:
+            message = "Isolate encountered an internal error."
```

Comment:
Can I suggest a `list + "\n".join` approach instead? That looks a bit messy and doesn't properly handle edge-cases eg. when both std fds are empty.

Reply:
Done!